src/runmode-dpdk.c

```diff
@@ -807,6 +816,13 @@ static int ConfigLoad(DPDKIfaceConfig *iconf, const char *iface)
     if (retval < 0)
         SCReturnInt(retval);
 
+    retval = ConfGetChildValueBoolWithDefault(
```

Comment:
if the function is void then retval should not be assigned. Then you don't need to check for the return code of retval (if-statement at L823)